### PR TITLE
Toolkit: Fix bug with rootUrls not being properly parsed when signing a private plugin.

### PR DESCRIPTION
**Pull request by dessen-xu**
**What this PR does / why we need it**:

The dependency of @grafana/toolkit commander was updated to 8.x in @grafana/toolkit 8.3.0, and the cmd args changed.

image

**Which issue(s) this PR fixes**:

https://github.com/grafana/grafana/issues/42860

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-toolkit/src/cli/index.ts [hunk 1]
```diff
@@ -200,7 +200,7 @@ export const run = (includeInternalScripts = false) => {
   program
     .command('plugin:sign')
     .option('--signatureType <type>', 'Signature Type')
-    .option('--rootUrls <urls...>', 'Root URLs')
+    .option('--rootUrls <type>', 'Root URLs')
```
Comment (mckn) [inline packages/grafana-toolkit/src/cli/index.ts:203]:
```suggestion
    .option('--rootUrls <url...>', 'Root URLs', function(url: string, urls: string[]) {
      if (typeof url !== 'string') {
        return urls;
      } 
      const parts = url.split(',');
      urls.push(...parts);
      return urls;
    }, [])
```

**Review by mckn — CHANGES_REQUESTED** [review]
This is great! Thanks for opening this. However the `<urls...>` added support for using the toolkit the following ways:

`grafana-toolkit plugin:sign --signatureType private --rootUrls https://example.com https://plugin.example.com`

`grafana-toolkit plugin:sign --signatureType private --rootUrls https://example.com --rootUrls https://plugin.example.com`

So if we remove the `<urls...>` it will break for everyone using it that way. Instead I suggested to add a custom option processor to support all scenarios mentioned above but also the:

`grafana-toolkit plugin:sign --signatureType private --rootUrls https://example.com,https://plugin.example.com`

**Comment by dessen-xu** [comment]
I get it, thanks

**Comment by dessen-xu** [comment]
I get it.

**Comment by dessen-xu** [comment]
done

**Review by mckn — APPROVED** [review]
LGTM! Great work.

**Comment by dessen-xu** [comment]
Thank you very much

**Pull request merged**